extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -147,8 +147,8 @@ export class AzureAccountProviderService implements vscode.Disposable {
 		const platform = os.platform();
 		const tokenCacheKey = `azureTokenCache-${provider.metadata.id}`;
 		const lockOptions = {
-			retryNumber: 100,
-			retryDelay: 50
+			retryNumber: 500,
```

Comment:
500 retries?!?! That seems incredibly high - is that normal for this persistence cache thing?

How do you know that 500 is going to be enough? Should this be something the user can control instead?

Reply:
500 is the default as well - and with MSAL-JS, its needed as otherwise time flies too quickly and token is not retrieved - leading to issues on frontend. I increased the delay as well for the same reason - 100 is the default for this - but giving some more time gap before retrying will reduce possibilities of missing retrieving access token.